index.html

```diff
@@ -14,6 +14,40 @@
     <!-- 3rd-party: FB -->
 
     <script src="./js/pages/home.js" defer></script>
```

Comment:
Sitas kodas turi buti po html kodo, kitaip jei JS ilgai galvos, vartotojas nieko ekrane nematys. Šitą dalį nukelti į apačia reikėtų